cvat/apps/engine/ddln/inventory_client.py

```diff
@@ -100,22 +110,55 @@ def record_task_creation(self, task_name, assignment_data):
         response_data = request.execute()
         return response_data['updates']['updatedRange']
 
+    def record_task_validation(self, task_name, validator, validation_date):
+        start_row, end_row = self._get_task_range(task_name)
+        if end_row < start_row:
+            raise ValueError("Records for task {!r} are not found.".format(task_name))
+        validation_date = "{:%d.%m.%Y}".format(validation_date)
+        rows = [[validator, validation_date] for _ in range(start_row, end_row + 1)]
```

Comment:
is it the same as `[validator, validation_date] * (start_row - end_row +1)` ?

Reply:
Yes, it's the same as `[[validator, validation_date]] * (end_row + 1 - start_row)`.

But it should be used with care for mutable elements (like in this case) - created list will have multiple pointers referring the same object. So manipulating any item (`rows[0].append(13)` for example) in the list will affect all the items in the list.

Also, you told me not to use python-specific features :smile: .